charts/ai-software-templates/chatbot/templates/application-gitops-job.yaml

```
@@ -12,7 +12,78 @@ spec:
     spec:
       containers:
         - name: repo-creator
-          image: quay.io/redhat-ai-dev/helm-chart-application-gitops:latest
+          command:
+            - /bin/bash
+            - -c
+            - |
+              #!/usr/bin/env bash
+              cd /tmp || exit
+              wget https://github.com/cli/cli/releases/download/v2.62.0/gh_2.62.0_linux_amd64.tar.gz
+              gunzip gh_2.62.0_linux_amd64.tar.gz
+              tar xf gh_2.62.0_linux_amd64.tar
+              export PATH=./gh_2.62.0_linux_amd64/bin:$PATH
+              export SOURCE_REPO_APP_CONTENT_PATH="chatbot"
+              export SOURCE_TEKTON_REPO_CONTENT_PATH="pac/pipelineRuns"
+              export TEKTON_FILE_APP_NAME_REPLACEMENT="application_name_replace"
+              export TEKTON_FILE_APP_NAMESPACE_REPLACEMENT="application_namespace_replace"
+              export TEKTON_FILE_QUAY_ACCOUNT_REPLACEMENT="quay_account_replace"
+              echo APP_NAME is "$APP_NAME"
+              echo APP_NAMESPACE is "$APP_NAMESPACE"
+              echo GITHUB_ORG_NAME is "$GITHUB_ORG_NAME"
+              echo GITHUB_SOURCE_REPO is "$GITHUB_SOURCE_REPO"
+              echo GITHUB_TETKON_SOURCE_REPO is "$GITHUB_TETKON_SOURCE_REPO"
+              echo GITHUB_DEFAULT_BRANCH is "$GITHUB_DEFAULT_BRANCH"
+              echo QUAY_ACCOUNT_NAME is "$QUAY_ACCOUNT_NAME"
+
+              export GH_TOKEN=$GITHUB_TOKEN
+              export GH_REPO="github.com/$GITHUB_ORG_NAME/$APP_NAME"
+
+              export HOME=/tmp
+
+              gh auth login -p https
+              gh auth setup-git --hostname=github.com
+
+              gh repo view "$GH_REPO" 2> /dev/null
```

```suggestion
              gh repo view "$GH_REPO" --json id 2> /dev/null
```
This should still give the pass/fail return code that you need without having to pull back so much information which by default is the description and the entire README.md

---

cool / yep